contrib/blueflood-docker/configs/ES-Setup/index_settings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
@ChandraAddala I know, but I wanted the blueflood-docker repo to be independent of other packages.